### DISCUSSION TITLE
Update the content of a toast

### DISCUSSION THREAD

**Discussion by rowanwernham** [calintamas/react-native-toast-message#207] [Q&A]
Is there a supported way to change the content of a toast message after it has been created?

I.e. - a toast shows with a loading icon + 'message sending', then updates with a check and 'message sent!'

Similar to what is in a library like this (each toast can have a key): https://3x.ant.design/components/message/

**Comment by appsgenie** [comment]
Did you find a solution for this?

**Comment by bruteforks** [comment]
Great idea. If anyone implements please let us know!
